### DISCUSSION TITLE
The ical4j Wiki is gone

### DISCUSSION THREAD

**Discussion by schube** [ical4j/ical4j#778] [Q&A]
**Describe the bug**
The wiki, which should be available at https://github.com/ical4j/ical4j/wiki is gone.

**To Reproduce**
Open https://github.com/ical4j/ical4j/wiki
You are then redirected to https://github.com/ical4j/ical4j

**Expected behavior**
See the wiki :-)

**Comment by benfortuna** [comment]
Haha, sorry about that. I posted an update here:

http://ical4j.github.io/2020/07/23/ical4j-user-guide.html

For now I've copied all the wiki info into the user guide and will work to improve it over time. (Feel free to fork and suggest edits).

**Comment by derTobsch** [comment]
in the user guide is a link to the wiki. Maybe delete that link :)
